example/cli/src/app/actions/actions.component.ts

```diff
@@ -16,6 +16,14 @@ const actionMapping:IActionMapping = {
       $event.shiftKey
         ? TREE_ACTIONS.TOGGLE_ACTIVE_MULTI(tree, node, $event)
         : TREE_ACTIONS.TOGGLE_ACTIVE(tree, node, $event);
+    },
+    mouseover: (tree, node, $event) => {
```

Comment:
Can you change to mouseOver and mouseOut to be consistent with the other names?

Reply:
"mouseover" and "mouseout" are angular's default event binding. The spelling is just like that (no camelcase). I prefer not to change that default naming to keep consistency.
However, I saw you also camelcase "dragOver". If that's the rule in this repository. I will still follow it.

Btw, there are also mouseenter and mouseleave (which events are not bubbling to children elements)

https://gist.github.com/AlexAegis/8741eb49db8112e6d0c05328d64a5e24

Reply:
Yeah, I prefer to be consistent with the other actions. As these are not DOM events, they don't need to be consistent with Angular's event binding. In retrospect, it might have been better to keep it lowercase.
Thanks